src/pybind/mgr/cephadm/tests/test_cephadm.py

```diff
@@ -951,3 +952,31 @@ def test_upgrade_run(self, use_repo_digest, cephadm_module: CephadmOrchestrator)
                 assert image == 'image@repo_digest'
             else:
                 assert image == 'image'
+
+    @mock.patch("cephadm.module.CephadmOrchestrator._run_cephadm")
+    def test_ceph_volume_no_filter_for_batch(self, _run_cephadm, cephadm_module: CephadmOrchestrator):
+        _run_cephadm.return_value = ('{}', '', 0)
+
+        error_message = """cephadm exited with an error code: 1, stderr:/usr/bin/podman:stderr usage: ceph-volume inventory [-h] [--format {plain,json,json-pretty}] [path]/usr/bin/podman:stderr ceph-volume inventory: error: unrecognized arguments: --filter-for-batch
+Traceback (most recent call last):
+  File "<stdin>", line 6112, in <module>
+  File "<stdin>", line 1299, in _infer_fsid
+  File "<stdin>", line 1382, in _infer_image
+  File "<stdin>", line 3612, in command_ceph_volume
+  File "<stdin>", line 1061, in call_throws"""
```

Comment:
this is a really fragile test.. it will break as soon as a line number changes

Reply:
no it doesn't. we're not actually running the cephadm binary here. This is just the error as it would be generated by `_run_cephadm`.  I'm only interested in the `if 'unrecognized arguments: --filter-for-batch' in str(e):` line